src/core/lib/security/authorization/rbac_translator.cc

```diff
@@ -282,16 +282,14 @@ absl::StatusOr<Rbac::Policy> ParseRule(const Json& json,
       }
       auto peer_or = ParsePeer(object.second);
       if (!peer_or.ok()) return peer_or.status();
-      principals =
-          absl::make_unique<Rbac::Principal>(std::move(peer_or.value()));
+      principals = std::move(peer_or.value());
```

Comment:
This can use `*peer_or` instead of `peer_or.value()`, since you've already checked that it is not an error above.

Reply:
Resolved all comments. PTAL.